docs/configuration.asciidoc

```diff
@@ -66,6 +66,23 @@ ElasticAPM::Sinatra.start(
 
 See <<getting-started-rack>>.
 
+[float]
+=== Grape and Rack
+
+When using APM with Grape and Rack you can configure it when starting
```

Comment:
Do we want to be explicit about that this is only when you use Grape _without_ Rails? I might be over thinking it.

Reply:
No harm in being over-explicit. I've added a "(without Rails)" detail.